sass/cd/cd-layout/_cd-layout.scss

```diff
@@ -248,10 +248,8 @@ body {
 
   .cd-layout-sidebar-first {
     margin-top: 1rem;
-
-    // I've styled this as a block with padding and bg colour.
     padding: 1rem;
-    background-color: var(--cd-blue-grey--light);
+    background-color: var(--brand-grey);
```

Comment:
This should be `--cd-blue-grey--light` but I understand the intention is to replace with brand variables where possible. 

This might be a contrast issue waiting to happen (which is why it's set to `--light`) if the `--brand-grey` is too dark. Since this is a sidebar, I think we can expect links and text to appear on top. No action needed, maybe just something to watch for in specific implementations depending on their value of `--brand-grey`